app/svelte/src/server/wrapInitialConfig.js

```diff
@@ -0,0 +1,19 @@
+export default config => ({
+  ...config,
+  module: {
+    ...config.module,
+    rules: [
+      ...config.module.rules,
+      {
+        test: /\.svelte$/,
+        loader: require.resolve('svelte-loader'),
+        options: {},
```

Comment:
can we also support `.html` files here?

Reply:
We can. I'm wondering if they will match other loaders elsewhere in the webpack config, haven't looked into these overrides enough yet.

Reply:
I just tested.. and putting this:
```js
test: /\.(svelte|html)$/,
```
works without causing any issues with other loaders.

Maybe check it out from your side.

Reply:
Thanks for checking that out. Will do.